Makefile

```
@@ -78,13 +93,13 @@ main: examples/main/main.cpp ggml.o whisper.o
 	./main -h
 
 ggml.o: ggml.c ggml.h
-	$(CC)  $(CFLAGS)   -c ggml.c
+	$(CC)  $(CFLAGS) -c ggml.c -o ggml.o
```

```suggestion
	$(CC)  $(CFLAGS)   -c ggml.c -o ggml.o
```

I really want to keep this whitespace 😄 
It makes the commands aligned vertically